### ISSUE TITLE
Apply spline to tetrahedral elements?

### ISSUE THREAD

**Issue by lucascbarbosa** [sfepy/sfepy#722]
I'm using a volumetric (tetrahedrical) mesh from a .vtk file, and I'd like to know if it's possible to apply a spline interpolation to the tet's edges... I dug into the `splinebox` class but didn't find anything related to 3D. Somebody could help me out here? thx

**Comment by rc** [comment]
No, curved boundary elements are not supported (except in IGA, which is however limited to single patch domains).

**Comment by lucascbarbosa** [comment]
What exactly are single patch domains?

**Comment by rc** [comment]
A domain described by a single NURBS patch, i.e. mapable to a line/rectangle/box.

**Comment by rc** [comment]
Closing answered questions with no further reaction...
